node/src/service_nimbus.rs

```diff
@@ -0,0 +1,634 @@
+// Copyright 2020-2022 Manta Network.
+// This file is part of Manta.
+//
+// Manta is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Manta is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Manta.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Nimbus-based Parachain Node Service
+
+use crate::rpc;
+// use codec::Codec;
+// use core::marker::PhantomData;
+use cumulus_client_cli::CollatorOptions;
+use cumulus_client_consensus_common::ParachainConsensus;
+// use cumulus_client_consensus_common::{
+//     ParachainBlockImport, ParachainCandidate, ParachainConsensus,
+// };
+use cumulus_client_network::BlockAnnounceValidator;
+use cumulus_client_service::{
+    prepare_node_config, start_collator, start_full_node, StartCollatorParams, StartFullNodeParams,
+};
+use cumulus_primitives_core::ParaId;
+// use cumulus_primitives_parachain_inherent::{
+//     MockValidationDataInherentDataProvider, MockXcmConfig,
+// };
+use cumulus_relay_chain_inprocess_interface::build_inprocess_relay_chain;
+use cumulus_relay_chain_interface::{RelayChainError, RelayChainInterface, RelayChainResult};
+use cumulus_relay_chain_rpc_interface::RelayChainRPCInterface;
+use jsonrpsee::RpcModule;
+use polkadot_service::{CollatorPair, NativeExecutionDispatch};
+
+// use futures::lock::Mutex;
+// use dolphin_runtime::RuntimeApi;
+pub use manta_primitives::types::{AccountId, Balance, Block, Hash, Header, Index as Nonce};
+
+// use cumulus_client_consensus_relay_chain::Verifier as RelayChainVerifier;
+use nimbus_consensus::{BuildNimbusConsensusParams, NimbusConsensus};
+use session_key_primitives::NimbusId;
+// use sc_client_api::ExecutorProvider;
+// use sc_consensus::{
+//     import_queue::{BasicQueue, Verifier as VerifierT},
+//     BlockImportParams,
+// };
```

Comment:
a bunch of forgotten commented code

Reply:
oof, that `service_nimbus` file snuck in from the very first pre-v3.3.0 code state on one of the merges. It's not used anywhere and shouldn't exist, deleting